packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -86,7 +86,63 @@ const getReadOnly = (
 }
 
 const ReflectionCard = (props: Props) => {
-  const {meeting, reflection, isClipped, openSpotlight, stackCount, showReactji, dataCy} = props
+  const {meetingRef, reflectionRef, isClipped, openSpotlight, stackCount, showReactji, dataCy} =
+    props
+  const reflection = useFragment(
```

Comment:
`reflection` is also taken 😁

Reply:
aha, if so, it would be better to just rename the return value of `useRef` to `reflectionReference`. 😁